libs/editor/egui_editor/include/egui_editor.h

```diff
@@ -0,0 +1,31 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef struct ThemedColor ThemedColor;
+
+
+
+
+
```

Comment:
Is this really generated?

Reply:
lol it is, but it shouldn't be committed